_includes/generator-form.html

```diff
@@ -0,0 +1,63 @@
+{::options parse_block_html="false" entity_output="as_input" /}
+<div class="tgmpa-generator-form">
+	<form id="generator-form" method="POST">
+		<input type="hidden" name="tgmpa_generate" value="1" />
+
+		<section class="tgmpa-form-primary">
+			<label for="tgmpa-flavor">I'm going to use TGMPA in a:<span class="required">*</span></label>
```

Comment:
For accessibility, don't make this a label, since it has no direct field, but use a legend inside of the fieldset instead. See http://webaim.org/techniques/forms/controls#radio as an example.